app/controllers/api/v1/quotes_controller.rb

```diff
@@ -20,6 +20,12 @@ def raw_params
   end
 
   def quoted_person
-    Person.find_or_create_by(full_name: raw_params[:quoted_person])
+    return unless raw_params[:quoted_person]
+    person = raw_params[:quoted_person]
+    if person[0] == '@'
```

Comment:
I think `.starts_with?('@')` might be more readable here. Also this is very inconsiderate to people whose real names start with `@`.